examples/activities/buildvrt.js

```diff
@@ -1,14 +1,15 @@
 "use strict";
 
 var fs = require('fs'),
-    path = require('path');
+    path = require('path'),
+    util = require("util");
 
 var tmp = require('tmp');
 
-var output = require("./output"),
-    shell = require("./shell");
+var output = require("../../lib/output"),
+    shell = require("../../lib/shell");
```

Comment:
Following along from externalization, we should move these into a `swfr-helpers` package or so.